src/HavenoClient.test.ts

```diff
@@ -2604,6 +2613,10 @@ async function executeTrade(ctxP: Partial<TradeContext>): Promise<string> {
     if (ctx.isStopped) return ctx.offerId!;
     if (!ctx.concurrentTrades) await testAmountsAfterComplete(ctx);
 
+    const tradeStatisticsPost = await arbitrator.getTradeStatistics();
```

Comment:
`executeTrade()` can be called simultaneously throughout the tests, so measuring the difference here won't be reliable.

It should be similar to your first submission, just moved into "Can complete a trade within a range", which tests completion of a single test.